### DISCUSSION TITLE
Helm Repository under github pages

### DISCUSSION THREAD

**Discussion by varet80** [fluent/fluent-operator#506] [Ideas]
Good day,

We are really happy with the new fluent operator. We would like to also have some helm repository, which every chart is released there.

A typical setup is: 
- Github pages with the repository
- Github Release of the Chart

This can be done with github actions.

I am happy to contribute in a change like this, if this is an interesting addition to repository

**Comment by wenchajun** [comment]
I think it's a great idea and look forward to your PR.

**Comment by benjaminhuo** [comment]
@varet80 we're submitting fluent-operator charts to fluent's official helm repo https://github.com/fluent/helm-charts/pull/299 now, hope it'll be accepted soon. @patrick-stephens @agup006

**Reply by patrick-stephens** [reply]
This is done now I think.

**Reply by benjaminhuo** [reply]
Yes, it's done already
